packages/runner-shared/src/container/container.spec.jsx

```diff
@@ -117,43 +136,19 @@ describe('<Container />', () => {
     beforeEach(() => {
       props = createProps()
       props.state.automation = automation.CONNECTED
-      props.util.hasSpecFile.returns(false)
+      props.hasSpecFile.returns(false)
       component = shallow(<Container {...props} />)
     })
 
     it('renders <NoSpec /> with config', () => {
       expect(component.find(NoSpec)).to.have.prop('config', props.config)
     })
 
-    it('renders the automation element', () => {
-      expect(component.find(`#${automationElementId}`)).to.exist
-    })
-
     it('renders the app when hash changes with and has a spec file', () => {
-      props.util.hasSpecFile.returns(true)
+      props.hasSpecFile.returns(true)
       component.find(NoSpec).prop('onHashChange')()
       component.update()
       expect(component.find(App)).to.exist
     })
   })
-
-  describe('when automation is connected and there is a spec file', () => {
-    let props
```

Comment:
This test was not doing anything very useful and was very brittle so I just nuked it 🤷